### DISCUSSION TITLE
How to ignore certain files while running Alt+c etc?

### DISCUSSION THREAD

**Discussion by rohitkrishna094** [PatrickF1/fzf.fish#225] [Q&A] [answered]
According to documentation `Alt +c` is used to cd into directory. But let's say that I have a folder which I want to ignore(think node_modules for example). Is there a way to do that in fzf.fish?

**Comment by PatrickF1** [comment, marked as answer]
Wrong repo? We don't have anything for cd. Maybe you're referring to the search directory features?

**Comment by rohitkrishna094** [comment]
Oh yeah my bad, I thought `Alt + c` worked because of fzf.fish but it was a feature of fzf itself. Thanks.

**Reply by PatrickF1** [reply]
Right, that comes from the shell integration that ships with fzf.
